common/autoware_ad_api_specs/include/autoware_ad_api_specs/vehicle.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 TIER IV, Inc.
```

Comment:
Better to make this the latest?
```suggestion
// Copyright 2023 TIER IV, Inc.
```

Reply:
updated [7c12946](https://github.com/autowarefoundation/autoware.universe/pull/2930/commits/7c1294671f974715967f10b73426e219bbcafdf0)